recipes/fastjet/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "fastjet" %}
+{% set version = "3.4.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastjet-{{ version }}.tar.gz
+  sha256: 86dea05ce7a030db9fe14e5bf1c0bc46351b6e439f9b9ef6b14a55ceefcc7b7a
+  patches:
+    - 0001-Don-t-overwrite-external-CXXFLAGS-and-LDFLAGS.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  missing_dso_whitelist:
+  # https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#whitelisting-shared-libraries
+  # All are under lib/python3.*/site-packages/fastjet/_fastjet_core/lib
+    - $RPATH/libfastjet.so.0
+    - $RPATH/libfastjettools.so.0
+    - $RPATH/libsiscone.so.0
+    - $RPATH/libsiscone_spherical.so.0
+    - $RPATH/libfastjetplugins.so.0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - libtool
+    - automake
+    - autoconf
+    - swig
+    - patch
+    - gmp  # In 'build' over 'host' as avoids 'ld: cannot find -lgmp: No such file or directory' error
```

Comment:
This definitely needs to be in `host` as otherwise your binaries will be broken when cross-compiling.

Reply:
Should be resolved now in https://github.com/lgray/staged-recipes/pull/4.